### PR TITLE
Add SMB to ztag

### PR DESCRIPTION
**Pull request by ajholland**
I'm fairly positive I did this correctly, if someone could check my work I'd appreciate it.

### CODE REVIEW

**Comment by ajholland** [comment]
^ That won't pass until definitions is updated

**Comment by cdzombak** [comment]
https://github.com/zmap/zgrab/pull/277 , which makes this change in zgrab output, was merged last week.

**Comment by dadrian** [comment]
> a new censys-defintions

I mean a new zsearch-defintions

File: ztag/schema.py [hunk 1]
```diff
@@ -1206,6 +1211,11 @@ class LintBool(String):
                     #"ssl_2": ztag_sslv2, # XXX
                 })
             }),
+            Port(445):SubRecord({
+                "smb":SubRecord({
+                    "smbv1_support":ztag_smb
```
Comment (dadrian) [inline ztag/schema.py]: This should be
```
   Port(445):SubRecord({
                  "smb":SubRecord({
                      "banner":ztag_smb
```

**Pull request merged**